zerver/tests/test_openapi.py

```diff
@@ -322,18 +324,19 @@ def get_standardized_argument_type(self, t: Any) -> type:
             else:  # nocoverage # in python3.5
                 args = t.__args__
             for st in args:
-                subtypes.append(self. get_standardized_argument_type(st))
+                subtypes.append(self.get_standardized_argument_type(st))
             return self.get_type_by_priority(subtypes)
-        elif origin == List:
-            return list
-        elif origin == Iterable:
-            return list
-        return self. get_standardized_argument_type(t.__args__[0])
+        elif origin in [List, Iterable]:
+            subtypes = [self.get_standardized_argument_type(st) for st in t.__args__]
+            return (list, self.get_type_by_priority(subtypes))  # type: ignore # this might be a Tuple[Type[List[Any]], Union[type, Tuple[type, type]]] but that means that it's a Tuple[type, Union[type, Tuple[type, type]]] too.
```

Comment:
But the ignored error here is

    got "Tuple[Type[List[Any]], Union[type, Tuple[type, type]]]", expected "Union[type, Tuple[type, type]]"

which says nothing about the `Tuple[type, Union[type, Tuple[type, type]]]` mentioned in the comment. Indeed, the type of this function is actually wrong:

```pycon
>>> zerver.tests.test_openapi.OpenAPIArgumentsTest().get_standardized_argument_type(List[List[int]])
(list, (list, int))
```

is neither `type` nor `Tuple[type, type]`.

Reply:
Oh, indeed, you're correct, @andersk. Thanks for pointing it out and fixing it! I seem to have gotten a little sloppy with the types towards the end.